var/spack/repos/builtin/packages/r/package.py

```diff
@@ -60,7 +60,7 @@ class R(AutotoolsPackage):
     depends_on('bzip2')
     depends_on('libtiff')
     depends_on('jpeg')
-    depends_on('cairo')
+    depends_on('cairo+pdf')
     depends_on('cairo+X', when='+X')
```

Comment:
oehm .. does this now need to be `cairo+X+pdf`?

Reply:
Nope. Spack merges all dependency constraints, it doesn't overwrite them.